py/torch_tensorrt/dynamo/conversion/impl/condition/ops.py

```diff
@@ -30,73 +29,28 @@ def where(
     x_shape = list(input.shape)
     y_shape = list(other.shape)
     condition_shape = list(condition.shape)
+    max_shape_len = max(len(x_shape), len(y_shape), len(condition_shape))
 
-    output_shape = list(torch.broadcast_shapes(condition_shape, x_shape, y_shape))
-
-    # expand shape
     if not isinstance(condition, TRTTensor):
         assert condition.dtype in (torch.bool, np.bool_), "condition dtype is not bool"
```

Comment:
If `condition.dtype` in (torch.bool, np.bool_), is it possible to convert to a supported type for `condition` so that it will not error out? because our goal is not to throw out errors if pytorch supports it. If not, I think a capability validator is needed.

Reply:
I saw uint8 type of condition was available for torch. Added type conversion.

Reply:
looks good to me. It would be better if add a suffix for the name here: https://github.com/pytorch/TensorRT/pull/2853/files#diff-84205b80e84f9b477f7b0b6dfaab7ea72f8426cc2673e240d2221401fe74c14cR41